src/Qir/Samples/StandaloneInputReference/CMakeLists.txt

```diff
@@ -34,7 +34,7 @@ add_test(
 if(DEFINED ENV{NATIVE_SIMULATOR})
     set(TEST_DEPS1 $ENV{NATIVE_SIMULATOR})
 else()
-    set(TEST_DEPS1 "${PROJECT_SOURCE_DIR}/../../Simulation/native/build/$ENV{BUILD_CONFIGURATION}")
+    set(TEST_DEPS1 "${PROJECT_SOURCE_DIR}/../../Simulation/native/build/drop")
```

Comment:
Will this result in multiple build configurations mixed together?

Reply:
No.
Previously the native simulator (NS) was built with GCC on Mac and Linux, the GCC was putting the binaries (of NS and its tests) to 
`Simulation/native/build`, the subsequent script lines were copying the binaries to 
`Simulation/native/build/drop`.
But on Windows the MSVC was putting the binaries to 
`Simulation/native/build/Release` (all the CI pipelines build the _Release config_ of all, the only pipeline that builds Debug config is the [`microsoft.qsharp-runtime.sanitized`](https://dev.azure.com/ms-quantum-public/Microsoft%20Quantum%20(public)/_build?definitionId=34&_a=summary) pipeline (running on Mac and Linux only), but still the Release and Debug binaries are never generated on the same machine).

Now the NS is built with Clang on all platforms, and Clang puts the binaries to `Simulation/native/build` on all platforms (and the subsequent script lines still copy the binaries to `Simulation/native/build/drop`).